CONTRIBUTING.md

```diff
@@ -0,0 +1,116 @@
+# Contributing Guide
+
+Contributions are always welcome!  
+If you have discovered a bug or have a feature request, [please create an issue on GitHub](https://github.com/kintone-labs/kintone-ui-component/issues/new/choose).  
+Pull requests are also welcome when you find trivial bugs or typos. 
+
+We use English for all commit messages, code comments, issues, pull requests.  
+
+Please follwing [Note for Naming Rule](#note-for-naming-rule).  
+
+
+This guide to develop and the explanation of the command options.
+
+## Command Guide
+
+The commands are providing as npm-critpts, please executing `npm run {{ Command }}` in the terminal.  
+
+|Command| Summary|
+| :---- | :---- |
+|build|build:umd(production) and build:esm|
+|build:esm|package and generate the ECMAScript Modules (ESM)|
+|build:umd|package and generate the production version of the umd file|
+|build:umd:dev|package and generate the development version of the umd file|
+|transpile|compile and build TypeScript code|
+|test|unit test|
+|eslint|eslint check|
+## How to develop
+### Clone and create branch
+
+#### 1.Please clone from GitHub repository at [here](https://github.com/kintone-labs/kintone-ui-component) to local
+
+```sh
+git clone git@github.com:kintone-labs/kintone-ui-component.git
+```
+
+#### 2.Check out to branch for dev
+Please refer [Branch naming rule](#Branch) and create a new branch(base on [branch:v0_dev](https://github.com/kintone-labs/kintone-ui-component/tree/v0_dev/)) for developing. 
+```sh
+git checkout <branch v0_dev>
+git branch <branch name>
+git checkout <branch name>
+```
+### Setup
+This repository is a Siggle-repo Monolith using npm.
```

Comment:
What is Siggle-repo Monolith??

Reply:
This means that there is only one project in this repository.
https://css-tricks.com/from-a-single-repo-to-multi-repos-to-monorepo-to-multi-monorepo/